File: server/scripts/inspect_excel.js
const XLSX = require('xlsx');
const path = require('path');

try {
    const workbook = XLSX.readFile('/home/zohra/Desktop/zohra-rms/zohra-rms-v2/Daily_Restaurant_Tracking.xlsx');
    console.log("Sheet Names:", workbook.SheetNames);

    if (workbook.SheetNames.includes('Summary')) {
        const sheet = workbook.Sheets['Summary'];
        const data = XLSX.utils.sheet_to_json(sheet, { header: 1 });
        console.log("\nSummary Sheet Data (First 20 rows):");
        console.log(data.slice(0, 20));
    }

    if (workbook.SheetNames.includes('Daily Transactions')) {
        const sheet = workbook.Sheets['Daily Transactions'];
        const data = XLSX.utils.sheet_to_json(sheet, { header: 1 });
        console.log("\nDaily Transactions Header:");
        console.log(data[0]);
    }

} catch (err) {
    console.error("Error reading file:", err);
}

File: server/scripts/inspect_excel.py

import pandas as pd

try:
    # Load the Excel file
    xl = pd.ExcelFile(
        "/home/zohra/Desktop/zohra-rms/zohra-rms-v2/Daily_Restaurant_Tracking.xlsx"
    )

    print("Sheet Names:", xl.sheet_names)

    if "Summary" in xl.sheet_names:
        df = xl.parse("Summary")
        print("\nSummary Sheet Head:")
        print(df.head(20).to_string())

        # Also let's check the 'Daily Transactions' columns just in case
    if "Daily Transactions" in xl.sheet_names:
        df_tx = xl.parse("Daily Transactions")
        print("\nDaily Transactions Columns:", df_tx.columns.tolist())

except Exception as e:
    print("Error reading Excel:", e)
